backend/app/Services/FileParser.php

```diff
@@ -211,6 +224,19 @@ private function remove($item)
     }
 
     /**
+     * Cancel the complete fetch and make a rollback of fetched files.
+     *
+     * @param $item
+     * @throws \Exception
+     */
+    private function abortParser($item)
+    {
+      DB::rollBack();
+
+      throw new \Exception("Failed to parse file '$item->name' with status '$item->status'. Please write a issue: https://github.com/devfake/flox/issues");
```

Comment:
Maybe dump $item object, so that the user could copy + paste that as well for the issue. As a developer, you want more infos about what went wrong.